ros2param/ros2param/api/__init__.py

```diff
@@ -140,16 +136,13 @@ def call_get_parameters(*, node, node_name, parameter_names):
     if response is None:
         e = future.exception()
         raise RuntimeError(
-            'Exception while calling service of node '
-            "'{args.node_name}': {e}".format_map(locals()))
+            f"Exception while calling service of node '{node_name}': {e}")
```

Comment:
Great example why f-strings are better: this bug was catched by the linters (before the string contained `args.node_name` but `args` isn't a variable in the current scope.

Same below in old line 170 / new line 162.